com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -124,6 +124,15 @@ private static Result exec(String... args) throws IOException {
         return exec(Arrays.asList(args));
     }
 
+    private static CompileExceptionError createCompileExceptionError(String message) {
```

Comment:
I decided to clean up how CompileExceptionErrors were created as I didn't get proper stack traces before